client/src/assets/icons/Under.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Under = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="200"
+    height="180"
+    strokeLinejoin="round"
+  >
+    <path stroke="#DDD" strokeWidth="25" d="M99,18 15,162H183z" />
+    <path stroke="#C00" strokeWidth="17" fill="#FFF" d="M99,18 15,162H183z" />
+    <path
+      d="M101,121a2,2 0 0,1-3,1l-2-3V103l-6,10-11-6l9-18h-6l-8,14
+l-3-3V97l8-14h20a4,4 0 0,1 4,4zm16,12l-41-34l1-3 42,35 8-10a5,5 0 0,1 9,0
+l22,29h-57l13-16zm-28-18l7,10 1,25a8,8 0 0,1-6-4l-1-17-8-11-18,31a4,4 0
+0,1-2-2l-1-6 17-32zm13-36a7,7 0 0,1 14,0a7,7 0 0,1-14,0"
+    />
```

Comment:
This looks like a formatting issue.
I would try to write `d` in a single line and let prettier handle it.

Reply:
solved!